css/resize/demos/resize-image/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Resize Image Playground — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    body {
+      min-height: 100vh;
+      padding: 50px;
+      display: flex;
+      align-items: flex-start;
+      justify-content: center;
+      background-color: #18191c;
+      color: #FFFFFF;
+      font-family: "Roboto", sans-serif;
+      font-size: 18px;
+    }
+
+    h2 {
+      margin-bottom: 16px;
+    }
```

Comment:
Привет! Спасибо большое за доку и за красивые интерактивные демки ❤ Котик невероятно милый 🥺

В этой демке всё супер, только убавим жирность:

```suggestion
    h2 {
      margin-bottom: 16px;
      font-weight: 500;
    }
```

Ещё можно добавить вот такое правило, тогда скролл покрасится в тёмные цвета:

```
    html {
      color-scheme: dark;
    }
```